www/php/verifysession.php

```diff
@@ -25,8 +25,7 @@
     exit;
 }
 
-$jwt_pk = file_get_contents(ROOT_DIR . IRMA_SERVER_PUBLICKEY);
-
+$jwt_pk = file_get_contents(IRMA_SERVER_URL . '/publickey');
```

Comment:
This is probably not what you want. In PHP this is run on every request. This means that you will not detect it when the public key changes unexpectedly. This means that the whole purpose of this public key, namely to verify the integrity of the result JWT, cannot be guaranteed anymore. You might want to reconsider this approach.

Reply:
makes sense. in what scenario is it appropriate to use this endpoint?